### PR TITLE
[bitnami/elasticsearch] Ability to add extraHosts

### PR DESCRIPTION
**Pull request by arpitjindal97**
**Description of the change**

Necessary if, 
 - You have another set of nodes that you want to introduce in this existing cluster.
 - Helps in migrating data to different K8s cluster by adding external data & master nodes
 - Extend an existing cluster running somewhere else

Parameter `extraHosts` is a list of manual configurable elasticsearch hosts.

**Benefits**
Environment variable `ELASTICSEARCH_CLUSTER_HOSTS` will get populated with these additional hosts

**Possible drawbacks**
N/A

**Applicable issues**
N/A

**Additional information**
N/A

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: elasticsearch
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch
   - https://www.elastic.co/products/elasticsearch
-version: 19.2.2
+version: 19.2.3
```
Comment (marcosbc) [inline bitnami/elasticsearch/Chart.yaml:28]: Since this is adding a new feature, could you increase the minor instead? I.e. to `19.3.0`.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the PR! Please have a look at my comment whenever you have a chance.

**Comment by arpitjindal97** [comment]
Do 'Rebase and Merge'

**Comment by marcosbc** [comment]
Hi @arpitjindal97, sorry for the delay, could you rebase your branch on your side with the latest changes? If I do it, the DCO check would not succeed, unfortunately.

**Comment by arpitjindal97** [comment]
Done

**Review by marcosbc — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**